src/components/Editor/SearchBar.js

```diff
@@ -396,8 +406,12 @@ const SearchBar = React.createClass({
   },
 
   buildPlaceHolder() {
-    if (this.state.functionSearchEnabled) {
-      return L10N.getStr("functionSearch.search.placeholder");
+    const { symbolSearchEnabled, selectedSymbolType } = this.state;
+    if (symbolSearchEnabled) {
+      return L10N.getFormatStr(
+        "symbolSearch.search.placeholder",
+        L10N.getStr(`symbolSearch.search.${selectedSymbolType}Placeholder`)
+      );
```

Comment:
@flodilo is it okay to interpolate strings like this:


*"Search for %S"*, where `%S` could be *function*, *variables*, ...